src/actions/UserActions.js

```diff
@@ -0,0 +1,25 @@
+import request from 'superagent-bluebird-promise';
+
+import {ACTION_TYPES} from './ActionTypes.js';
+import {busyToggle} from './appActions.js';
+import {buildUrl, catchHandler} from './fetch.js';
+
+export function authenticateUser(data) {
+    console.log('we in here \n' + data)
```

Comment:
Before creating a PR, check your code and delete everything that is not needed
- TODO's
- Commented out code that serves no purpose
- console.log()
- Code to try something that is no longer necessary or in use
- ...